### PR TITLE
Remove typecheck test and clean things up

### PR DESCRIPTION
**Pull request by sobolevn**

### CODE REVIEW

**Comment by sobolevn** [comment]
Refs https://github.com/typeddjango/django-stubs/pull/1552

**Comment by intgr** [comment]
You didn't remove the `typecheck_tests.py` script itself?

**Comment by sobolevn** [comment]
Good catch :)

File: scripts/git_helpers.py [hunk 1]
```diff
@@ -1,34 +0,0 @@
-import shutil
-from typing import Optional, Union
-
-from git.remote import RemoteProgress
-from git.repo import Repo
```
Comment (intgr) [inline scripts/git_helpers.py:5]: We can also get rid of the `gitpython` dependency.

**Review by intgr — COMMENTED** [review]
I feel a bit bad to delete all this code, people clearly went through a fair bit of effort to create this system.

Maybe we should keep it around until we have a new system to replace it? It hasn't really been a big maintenace burden, unless I'm missing something?

**Comment by intgr** [comment]
> It hasn't really been a big maintenace burden, unless I'm missing something?

Oh OK, I see the errors coming from mypy 1.4 update now. That seems like a compelling reason.

File: scripts/git_helpers.py (hunk 1 above)
Comment (sobolevn) [reply scripts/git_helpers.py:5]: > Maybe we should keep it around 

We have it: in our git history :)

> until we have a new system to replace it?

I am working on `stubtest` PR right now.

**Review by intgr — APPROVED** [review]
👍

**Pull request merged**